libtelemetry/src/main/java/com/mapbox/android/telemetry/VisionEventFactory.java

```diff
@@ -30,11 +32,32 @@ public Event createVisionEvent(Event.Type type) {
     return BUILD_EVENT_VISION.get(type).build();
   }
 
+  public VisionAttachment createVisionAttachment(String filePath, MediaType mediaType,
+                                                  AttachmentMetadata attachmentMetadata) {
+    return buildVisionAttachment(filePath, mediaType, attachmentMetadata);
```

Comment:
wouldn't this be a little more readable and less redundant if we do:
`return new VisionAttachment(attachmentMetadata, filepath, mediaType);`